README.md

```diff
@@ -74,11 +74,62 @@ NB: The integration tests destroy and re-create the given databases each time th
 
 We support PostgreSQL 9.4 and greater. The user passed in the postgres url needs to have full creation rights on the given database.
 
+## Tesing the entrypoint script together with the npm tests
+
+You can test both the code in the repository and the docker image entrypoint by runing the `docker-compose.test.yml` file. You first need to install the test submodules that are found in in the bash tests folder by running `git submodule update --init`.  Once done you can run the tests by running `docker-compose -f docker-compose.test.yml up -d` and checking the tests results through `docker logs cht-couch2pg_sut_1`
+
 
 ## Example usage
 
 You should probably install cht-couch2pg as a service and leave it to do its thing, as it should be able to run independently without any user input.
 
+### Running  the docker image
+
+You can run the [docker image](https://hub.docker.com/r/medicmobile/cht-couch2pg) avaialable on docker hub.  You will need to provide the required environment variables.  A sample docker-compose snipet is shown below. This image can work with the [cht-postgres](https://hub.docker.com/r/medicmobile/cht-postgres) docker image also available on dockerhub.
+
+```yaml
+
+  couch2pg:
+        container_name: cht_couch2pg
+        image: medicmobile/cht-couch2pg:test-rc.1
+        environment:
+           COUCHDB_URL: "http://${COUCHDB_USER:-cht}:${COUCHDB_PASSWORD:-cht_password}@medic-api:5988/medic"
+           COUCH2PG_SLEEP_MINS: '720'
+           COUCH2PG_DOC_LIMIT: '1000'
+           COUCH2PG_RETRY_COUNT: '5'
+           POSTGRES_DB: cht-postgres
+           COUCH2PG_CHANGES_LIMIT: 100
+           POSTGRES_USER_NAME: cht_couch2pg
+           POSTGRES_DB_NAME: cht
+           POSTGRES_PASSWORD: couch2pg_password
+           API_URL: http://medic-api:5988
+        depends_on:
+          - cht-postgres
+  cht-postgres:
+        container_name: cht-postgres
+        image: medicmobile/cht-postgres:release-postgres13-rc.1
+        environment:
```

Comment:
When I do testing using this. I expose the ports for postgres so that I can connect with PGAdmin.